--- a/packages/opentelemetry-instrumentation-transformers/tests/test_pipeline.py
+++ b/packages/opentelemetry-instrumentation-transformers/tests/test_pipeline.py
@@ -1,6 +1,5 @@
 import pytest
 import importlib.util
-from opentelemetry.sdk._logs import ReadableLogRecord
 from opentelemetry.semconv._incubating.attributes import (
     gen_ai_attributes as GenAIAttributes,
 )
@@ -92,7 +91,7 @@
     assert_message_in_logs(logs[1], "gen_ai.choice", choice_event)
 
 
-def assert_message_in_logs(log: ReadableLogRecord, event_name: str, expected_content: dict):
+def assert_message_in_logs(log, event_name: str, expected_content: dict):
     assert log.log_record.event_name == event_name
     assert (
         log.log_record.attributes.get(GenAIAttributes.GEN_AI_SYSTEM) == "transformers"
